test/xpu_api/utilities/utility/forward/move_if_noexcept.pass.cpp

```diff
@@ -46,7 +45,7 @@ struct MoveNotNoexceptCopy
 {
     MoveNotNoexceptCopy() = default;
 
-    MoveNotNoexceptCopy(MoveNotNoexceptCopy&&) noexcept(false) {};
+    MoveNotNoexceptCopy(MoveNotNoexceptCopy&&) noexcept(false){};

```

Comment:
Please roll back this change
```suggestion
    MoveNotNoexceptCopy(MoveNotNoexceptCopy&&) noexcept(false) {};
```